messenger.rst

```diff
@@ -514,12 +518,17 @@ for each bus looks like this:
 
 #. _Your own collection of middleware_;
 
-#. ``route_messages`` middleware. Will route the messages you configured to their
+#. ``send_message`` middleware. Will route the messages you configured to their
    corresponding sender and stop the middleware chain;
 
-#. ``call_message_handler`` middleware. Will call the message handler(s) for the
+#. ``handle_message`` middleware. Will call the message handler(s) for the
    given message.
 
+.. note::
+
+    These middleware names are actually shortcuts working by convention.
+    The real service ids are prefixed with the ``messenger.middleware.`` namespace.
+
 Adding your own Middleware
```

Comment:
It would make more sense to switch this section and the next one imho, so we:
- List the default middleware
- Explain how to disable them in _Disabling default Middleware_
- Switch to how to add your own (_Adding your own middleware_ and _Using Middleware factories_)

Instead of the current _default middleware list_ => _add you own_ => _back to default middleware_ => back to adding your own.